syntax = "proto2";

package st.planning;
// option go_package = "st.proto.ai/onboard/planning.scheduler/proto/smooth_reference_line";

message SmoothedReferenceLineResultProto {
  message LaneIdVec {
    repeated uint64 lane_id = 1;
  }
  repeated LaneIdVec lane_id_vec = 1;
}